rhel/matcher.go

```diff
@@ -19,7 +19,8 @@ func (*Matcher) Name() string {
 
 // Filter implements driver.Matcher.
 func (*Matcher) Filter(record *claircore.IndexRecord) bool {
-	return record.Distribution.DID == "rhel"
+	return record.Distribution != nil &&
```

Comment:
why this random change and only for rhel?

Reply:
It was the only instance I ran into while testing.

Reply:
makes me a bit skeptical that it maybe an issue somewhere else, maybe just confirm its not.